src/client/resources/assets/localization/en-us.json

```diff
@@ -282,7 +282,6 @@
   "minicraft.settings.mode.hardcore": "Hardcore",
   "minicraft.settings.mode.score": "Score",
   "minicraft.settings.scoretime": "Time (Score Mode)",
-  "minicraft.settings.screenshot_scale": "Screenshot Scale",
```

Comment:
You removed a localisation entry?

Reply:
I have removed too many lines. I have added it back now.